.travis.yml

```diff
@@ -5,11 +5,11 @@ node_js:
   - '0.12'
   - 'iojs-1'
   - 'iojs-2'
-  - '5',
-  - '6',
-  - '7',
-  - '8',
-  - '9',
+  - '5'
```

Comment:
I think we want to do node 4 and up, since that is the supported version by node.